.github/actions/regression-tests/action.yml

```diff
@@ -62,6 +62,19 @@ runs:
         fi
         curl -fsSL "https://download.docker.com/linux/static/stable/${ARCH}/docker-${DOCKER_VERSION}.tgz" | tar xz --strip 1 docker/docker -C ./docker
         registry="ghcr.io/romange" && for image in fedora:30 ubuntu:noble; do ./docker pull "$registry/$image"; done
+    - name: Free disk space
+      if: contains(runner.labels, 'self-hosted') == false
+      shell: bash
+      run: |
+        echo "===================Before freeing up space ============================================"
+        df -h
+        rm -rf /hostroot/usr/share/dotnet
+        rm -rf /hostroot/usr/local/share/boost
+        rm -rf /hostroot/usr/local/lib/android
+        rm -rf /hostroot/opt/ghc
```

Comment:
These paths are not mounted in `ioloop-v2-regtests` but this step is still run. Does  `/hostroot` need to be mounted in that ~~action~~ workflow?

Reply:
we should mount them, indeed. this mounts the host root disk, so we could free up some space.